src/store/audio.ts

```diff
@@ -2751,6 +2773,29 @@ export const audioCommandStore = transformCommandStore(
       },
     },
 
+    COMMAND_MULTI_SET_AUDIO_PAUSE_LENGTH_SCALE: {
+      mutation(
+        draft,
+        payload: {
+          audioKeys: AudioKey[];
+          pauseLengthScale: number;
+        },
+      ) {
+        for (const audioKey of payload.audioKeys) {
+          audioStore.mutations.SET_AUDIO_PAUSE_LENGTH_SCALE(draft, {
+            audioKey,
+            pauseLengthScale: payload.pauseLengthScale,
+          });
+        }
+      },
+      action(
+        { commit },
+        payload: { audioKeys: AudioKey[]; pauseLengthScale: number },
+      ) {
+        commit("COMMAND_MULTI_SET_AUDIO_PAUSE_LENGTH_SCALE", payload);
+      },
```

Comment:
最近ちょっと書き方変わったので追従（ちなみにコード補完されやすくなりました）
```suggestion
      action(
        { mutations },
        payload: { audioKeys: AudioKey[]; pauseLengthScale: number },
      ) {
        mutations.COMMAND_MULTI_SET_AUDIO_PAUSE_LENGTH_SCALE(payload);
      },
```